src/scene-entry-manager.js

```diff
@@ -77,6 +90,7 @@ export default class SceneEntryManager {
       return;
     }
 
+    this.scene.systems["motion-capture-replayer"].remove();
```

Comment:
ah, I was calling `remove()` here as a hack because it monkey patches the a-frame tracked controls component (and slows it down). i should reverse these. (ideally we just wouldn't include this system at all except on the bots, I don't know how to do that.)

Reply:
I would add a comment here to explain that.